charts/vsphere-csi/v2.1.0/values.yaml

```diff
@@ -1,29 +1,65 @@
 vCenter:
   host: ""
-  datacenters: ""
   port: 443
   insecureFlag: true
+  clusterId: ""
+  datacenters: ""
   username: ""
   password: ""
   configSecret:
     name: "vsphere-config-secret"
     generate: true
-    configTemplate: |
-      [Global]
-      cluster-id = {{ uuidv4 | quote }}
-      user = {{ .Values.vCenter.username | quote }}
-      password = {{ .Values.vCenter.password | quote }}
-      port = {{ .Values.vCenter.port | quote }}
-      insecure-flag = {{ .Values.vCenter.insecureFlag | quote }}
+  configTemplate: |
+    [Global]
+    cluster-id = {{ required ".Values.vCenter.clusterId must be provided" (default .Values.vCenter.clusterId .Values.global.cattle.clusterId) | quote }}
+    user = {{ .Values.vCenter.username | quote }}
+    password = {{ .Values.vCenter.password | quote }}
+    port = {{ .Values.vCenter.port | quote }}
+    insecure-flag = {{ .Values.vCenter.insecureFlag | quote }}
 
-      [VirtualCenter {{ .Values.vCenter.host | quote }}]
-      datacenters = {{ .Values.vCenter.datacenters | quote }}
+    [VirtualCenter {{ .Values.vCenter.host | quote }}]
+    datacenters = {{ .Values.vCenter.datacenters | quote }}
+
+csiController:
+  csiResizer:
+    enabled: false
+  image:
+    repository: gcr.io/cloud-provider-vsphere/csi/release/driver
+    tag: v2.1.0
+    pullPolicy: Always
+    csiAttacher:
+      repository: quay.io/k8scsi/csi-attacher
+      tag: v3.0.0
+    csiResizer:
+      repository: quay.io/k8scsi/csi-resizer
+      tag: v1.0.0
+    livenessProbe:
+      repository: quay.io/k8scsi/livenessprobe
+      tag: v2.1.0
+    vsphereSyncer:
+      repository: gcr.io/cloud-provider-vsphere/csi/release/syncer
+      tag: v2.1.0
+      pullPolicy: Always
```

Comment:
If we expose the pullPolicy for one image, should we expose the pullPolicy for all images?

Reply:
I'm assuming @PennyScissors based it on whatever the chart set, which is not needed to be changed for this release.

Reply:
removed